src/yoroi-wallets/walletManager.ts

```diff
@@ -697,3 +697,5 @@ class WalletManager {
 export const walletManager = new WalletManager()
 
 export default walletManager
+
+export const mockWalletManager = {} as WalletManager
```

Comment:
does it worth mocking in here?

Reply:
not sure, I want a mock walletManager somewhere